catalyst/core/callbacks/checkpoint.py

```diff
@@ -121,6 +121,7 @@ def __init__(
         resume_dir: str = None,
         metrics_filename: str = "_metrics.json",
         load_on_stage_end: str = None,
+        preload_best: bool = False,
```

Comment:
what do you think about `load_best_on_stage_start` name? it would be consistent with `load_on_stage_end` one

Reply:
Sure, sounds right, also need to add more comments about this argument.